src/pylint_datetime/__init__.py

```diff
@@ -88,88 +88,67 @@ def visit_call(self, node: nodes.Call) -> None:
                     self.add_message("datetime-timedelta-no-keyword-args", node=node)
                     break
 
+        if func_name in ("fromordinal", "fromisocalendar"):
+            if not self.has_replace_with_tzinfo(node):
+                self.add_message(
+                    "datetime-missing-timezone-replace", node=node, args=(func_name,)
+                )
+
         if func_name in ("datetime", "now", "fromtimestamp", "astimezone"):
             self.check_function_called_with_timezone(node, func_name)
 
         if func_name in ("today", "utcnow", "utcfromtimestamp", "utctimetuple", "time"):
             self.add_message("datetime-naive-object-used", node=node, args=(func_name,))
 
-    def naive_properties_methods_replace(
-        self, node: nodes.Assign, assigned_value: Any, assigned_var: Any
-    ) -> None:
-        """Check for a replace call specifiying timezone after naive property or function.
+    def visit_attribute(self, node: astroid.Attribute) -> None:
+        """Check attribute access to datetime.datetime.min/max, datetime.time.min/max.
 
         Args:
-            node: the node that visit_call originates from.
-            assigned_value: the value of the assignment that triggered function call.
-            assigned_var: the variable that the value was assigned to.
+            node: origin node for attribute.
         """
-        next_node = assigned_var
-        next_is_empty = True
-        if next_node and next_node.next_sibling():
-            next_is_empty = False
-            next_node = next_node.next_sibling()
-            if (
-                isinstance(next_node.value, astroid.Call)
-                and isinstance(next_node.value.func, astroid.Attribute)
-                and next_node.value.func.attrname == "replace"
-            ):
-                if not any(
-                    isinstance(arg, astroid.Keyword) and arg.arg == "tzinfo"
-                    for arg in next_node.value.keywords
-                ) and (
-                    not any(
-                        isinstance(arg, astroid.Attribute)
-                        for arg in next_node.value.args
-                    )
-                ):
-                    new_node = next_node.value
-                    self.add_message(
-                        "datetime-call-without-timezone",
-                        node=new_node,
-                        args=("replace",),
-                    )
-            else:
+        if not isinstance(node, astroid.Attribute):
+            print("ooh, hello")
+            return
+        if isinstance(node.expr, astroid.Name):
+            name = node.expr.name
+        elif isinstance(node.expr, astroid.Attribute):
+            name = node.expr.attrname
+        else:
+            return
+        if node.attrname in ("min", "max") and name in (
+            "datetime",
+            "time",
```

Comment:
We should probably check for fully qualified names (`datetime.datetime` / `datetime.time`), but this is very low prio, so please just create an issue for it and do not try to fix it in this PR.